### ISSUE TITLE
Wayland NVIDIA ignores VkCompositeAlphaFlagsKHR

### ISSUE THREAD

**Issue by GamesTrap** [GamesTrap/TRAP#101]
Currently the **proprietary** NVIDIA driver ignores the Vulkan `VkCompositeAlphaFlagsKHR::VK_COMPOSITE_ALPHA_OPAQUE_BIT_KHR` flag which is needed to show a non-transparent window while the `VkSurface` itself contains alpha information.

Current behavior:

![image](https://user-images.githubusercontent.com/16561717/222858464-a69bcfb6-a738-44c0-822f-a62c1f044190.png)

Expected behavior:

![image](https://user-images.githubusercontent.com/16561717/222858234-64fd8ecf-671e-47eb-bdea-006fbeb843ea.png)

https://github.com/NVIDIA/egl-wayland/pull/71 states that this should get fixed  for driver release 530 (_copium_).
[Issue on NVIDIA forum](https://forums.developer.nvidia.com/t/wayland-vulkan-ignores-vkcompositealphaflagskhr/202227/1)

**Comment by GamesTrap** [comment]
Worked around by replacing 
```cpp
TRAP::Graphics::RenderCommand::SetBlendConstant(TRAP::Graphics::BlendConstant::SrcAlpha,
                                                TRAP::Graphics::BlendConstant::OneMinusSrcAlpha);
```
with
```cpp
TRAP::Graphics::RenderCommand::SetBlendConstant(TRAP::Graphics::BlendConstant::SrcAlpha,
                                                TRAP::Graphics::BlendConstant::One,
                                                TRAP::Graphics::BlendConstant::OneMinusSrcAlpha,
                                                TRAP::Graphics::BlendConstant::OneMinusSrcAlpha);
```
everywhere.

**Comment by GamesTrap** [comment]
Fixed with NVIDIA driver 530.41.03
